### PR TITLE
Update manage-mde-post-migration-intune.md

### LINKED ISSUE #12674: Update steps
[Enter feedback here]

Kindly update the following steps under the sub heading - **Find your Microsoft Defender for Endpoint settings in Intune**

Go to the Azure portal (https://portal.azure.com/) and sign in. - _should now be go to Endpoint Manager/ Intune Admin Center_

Under Azure Services, choose Intune. - Not required

In the navigation pane on the left, choose Device configuration, and then, under Manage, choose Profiles.

Select an existing profile, or create a new one.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 0f198778-4f48-c530-84f5-3c9d8c812edb
* Version Independent ID: 2e88d353-72de-3a49-c536-90582ef6cc29
* Content: [Manage Microsoft Defender for Endpoint using Intune](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/manage-mde-post-migration-intune?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/manage-mde-post-migration-intune.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/manage-mde-post-migration-intune.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @denisebmsft
* Microsoft Alias: **deniseb**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation
Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12674

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/manage-mde-post-migration-intune.md [hunk 1]
```diff
@@ -38,13 +38,11 @@ We recommend using Microsoft Intune to manage your organization's threat protect
 > [!IMPORTANT]
 > You must have either the global administrator or service administrator role assigned in Intune to configure the settings described in this article. To learn more, see **[Types of administrators (Intune)](/mem/intune/fundamentals/users-add#types-of-administrators)**.
 
-1. Go to the Azure portal ([https://portal.azure.com](https://portal.azure.com)) and sign in.
+1. Go to the [Microsoft Intune Admin center](https://endpoint.microsoft.com/#home) and sign in.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/manage-mde-post-migration-intune.md]:
```suggestion
1. Go to the [Microsoft Intune admin center](https://endpoint.microsoft.com/#home) and sign in.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**